pkg/reconciler/pubsub/topic/topic.go

```diff
@@ -413,21 +234,6 @@ func (r *Reconciler) reconcilePublisher(ctx context.Context, topic *v1alpha1.Top
 	return nil, svc
 }
 
-func (r *Reconciler) getPublisher(ctx context.Context, topic *v1alpha1.Topic) (*serving.Service, error) {
```

Comment:
aren't we using this?

Reply:
seems we are not, we just do everything from the reconcilePublisher, discard the comment